src/custom-client-scripts/utils.js

```diff
@@ -24,7 +24,7 @@ export function setUniqueUrls (collection) {
 
 export function findProblematicScripts (collection) {
     const nonEmptyScripts              = collection.filter(s => !!s.content);
-    const scriptsWithDuplicatedContent = getDuplicatedScripts(nonEmptyScripts, s => s.content);
+    const scriptsWithDuplicatedContent = getDuplicatedScripts(nonEmptyScripts, s => s.content + s.page.toString());
```

Comment:
Looks like a hack. Script's content can be very large. This sum can lead to performance degradation and memory leaks, can we avoid it somehow?

Reply:
Theoretically you are right.  But I guess we should wait until the first customer issue, because we don't know now with which amount of client scripts customers will operate.

> can we avoid it somehow?
We can calculate hash of the script content and group by this.

Reply:
For me it would be better if we show a warning when we really inject the same script to a page with particular url. I mean, as far as I can see if we set the same page by url and by some RegExp we can't detect this 'overinjecting'. Correct me if I'm wrong.

Reply:
@helen-dikareva 

> For me it would be better if we show a warning when we really inject the same script to a page with particular url.

We show such warning - https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/api/es-next/custom-client-scripts/test.js#L41

Reply:
@AndreyBelym 

I will rewrite the duplicated script calculation using the hash. It's faster to do than discuss.

Reply:
I think that the current solution is quite enough, but I meant showing warning for the following case (in [my comment])(https://github.com/DevExpress/testcafe/pull/4118#discussion_r311535403)
```js
fixture `f`
    .page `https://devexpress.github.io/testcafe/documentation/getting-started/`;

test.clientScripts([
    {
        page:   'https://devexpress.github.io/testcafe/documentation/getting-started/',
        module: 'lodash'
    },
    {
        page:   /getting-started/,
        module: 'lodash'
    }
])('t1', async t => {
    await t.click('body');
});

```